bodhi/server/config.py

```diff
@@ -407,6 +407,12 @@ class BodhiConfig(dict):
         'initial_bug_msg': {
             'value': '%s has been submitted as an update to %s. %s',
             'validator': unicode},
+        'greenwave_api_url': {
+            'value': 'https://greenwave.fedoraproject.org/api/v1.0',
+            'validator': unicode},
+        'greenwave_enabled': {
+            'value': False,
+            'validator': _validate_bool},
```

Comment:
Nothing could ever have meaningfully used ```ci.enabled``` yet since it wasn't hooked up to anything, so I don't think we have a backwards incompatibility concern to worry about.